caesars-palace/blocks/header/header.js

```diff
@@ -129,6 +129,62 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
   }
 }
 
+const showMore = (nav) => {
+  const ul = nav.querySelector('.nav-sections > .local-nav > ul');
+  const more = document.createElement('li');
+  more.classList.add('more');
+  const moreText = document.createElement('div');
+  moreText.classList.add('more-text');
+  const aMore = document.createElement('a');
+  aMore.classList.add('more-link');
+  aMore.text = 'MORE';
+  const downArrow = document.createElement('div');
+  downArrow.classList.add('down-arrow');
+  aMore.appendChild(downArrow);
+  moreText.appendChild(aMore);
+  const dropdownMenu = document.createElement('div');
+  dropdownMenu.classList.add('dropdown-menu');
+
+  // We only allow nine menu items to remain in the header
+  while (ul.children.length > 9) {
```

Comment:
Could you explore using ES6 style array functions instead of a `while` loop? Could you do something like `[...ul.children]` and then a `forEach` that skips the first `n` indexes?

Reply:
+1. Make the `n` value a variable, too.